### DISCUSSION TITLE
[issue][jenkins] Devlake can't pull data from specific jenkins pipelines

### DISCUSSION THREAD

**Discussion by imostafa1980** [apache/devlake#7901] [General]
when we integrated devlake with our jenkins server , we found that some pieplines data can be pulled successfully to devlake and some devlake can't get data knowing that we use same connection and user permissions 
can you support please as i can reproduce the issue 
i uploaded to you jenkins /api/json of for one working piepline and jenkins /api/json of a non working piepleine 
aslo uploaded logs of both jobs (working and non working) from devlake with highset debug level 
if you can help me to know the reason why devlake job can't pull data from jenkins in some random pieplines 

i validate that no data pulled from both grafana and also on the mysql database levels no new data added to deployment related tables

[json-not-working-pipeline.json](https://github.com/user-attachments/files/16623416/json-not-working-pipeline.json)
[json-working-pipeleine.json](https://github.com/user-attachments/files/16623417/json-working-pipeleine.json)
[logging (6)-non-working.tar.gz](https://github.com/user-attachments/files/16623418/logging.6.-non-working.tar.gz)
[logging (7)-working.tar.gz](https://github.com/user-attachments/files/16623419/logging.7.-working.tar.gz)

**Comment by MohamedSamy74** [comment]
hello dears, I have the same issue too!!!
can you help with the investigation plz

**Comment by Startrekzky** [comment]
Hi @imostafa1980 and @MohamedSamy74 , 
1. What Jenkins server version are you on?
2. What DevLake version are you on?
3. Are there any differences between the working and not-working Jenkins pipelines? 

Besides, DevLake only collects Jenkins builds, not Jenkins pipeline runs. Could this be the reason?
